blocks/flip-cards/flip-cards.js

```diff
@@ -0,0 +1,26 @@
+export default function decorate(block) {
+  const ul = document.createElement('ul');
+  ul.classList.add('flip-cards-wrap');
+
+  [...block.children].forEach((row) => {
+    const [defaultState, hoverState] = row.children;
+
+    defaultState.classList.add('flip-cards-default');
+    hoverState.classList.add('flip-cards-hover');
+
+    hoverState.querySelectorAll('li a').forEach((link) => {
+      const chevron = document.createElement('span');
+      chevron.classList.add('icon', 'icon-chevron-right');
+      link.classList.add('button');
+      link.append(chevron);
```

Comment:
the links behave a little strange for me (chrome MacOS), they show the link cursor first but after a second switch back to the normal mouse cursor, clicking works, it is just a little irritating

Further I don't now if we should implement a dedicated solution for links in the flip card or rather use the default link behaviour we have everywhere else

Reply:
do you mean that hand (pointer) cursor is only visible the first time you hover over the links? I can't reproduce it.

> Further I don't now if we should implement a dedicated solution for links in the flip card or rather use the default link behaviour we have everywhere else
It was decided the link should have the dark plum color and add a chevron. @DaphneKoren has more infor about it.

Reply:
The link hover is a bit strange I can show you in a short screenshare, just ping me when you are available

Reply:
@DaphneKoren please reconsider the link requirements for this, in my opinion the links are quite hard to spot now as the only hint is the icon, on top I see no reason why link in the cards should look completely different in those cards as in normal text on the page where they are bold and different color

Reply:
@catarinajordao What do you suggest here?